test/BulletDynamics/test_deterministic_save_and_restore.cpp

```diff
@@ -0,0 +1,221 @@
+#include "../../Extras/Serialize/BulletFileLoader/bFile.h"
+#include <btBulletDynamicsCommon.h>
+#include "../../src/BulletDynamics/Featherstone/btMultiBodyDynamicsWorld.h"
+#include "../../src/BulletDynamics/Featherstone/btMultiBodyConstraintSolver.h"
+#include "../../Extras/Serialize/BulletWorldImporter/btMultiBodyWorldImporter.h"
+#include "../../Extras/Serialize/BulletWorldImporter/btBulletWorldImporter.h"
+#include "../../Extras/Serialize/BulletWorldImporter/btWorldImporter.h"
+#include "../../src/BulletCollision/CollisionDispatch/btCollisionObject.h"
+#include "../../src/BulletCollision/CollisionDispatch/btCollisionWorld.h"
+#include "LinearMath/btVector3.h"
+#include "LinearMath/btAlignedObjectArray.h"
+#include "../../src/BulletDynamics/Dynamics/btRigidBody.h"
+
+#include "../../examples/CommonInterfaces/CommonRigidBodyBase.h"
+#include <gtest/gtest.h>
+#define USE_MOTIONSTATE 0
```

Comment:
Why are we setting this #define in this test? It seems like it's not used to test anything, can we remove it?